en/mkdocs.yml

```diff
@@ -104,35 +104,44 @@ nav:
         - 'Validate JWT Based On JWKS ': guides/access-delegation/validate-jwt-based-on-jwks.md
         - 'Configure Microprofile JWT 1.0': guides/access-delegation/microprofile-jwt.md
       - 'Authorization' : guides/authorization/overview.md
-      - 'User Management' :
+      - 'Manage Users':
         - 'Overview': guides/user-mgt/user-mgt-overview.md 
         - 'Manage User Operations':
           - 'Overview': guides/user-mgt/manage-user-overview.md
           - 'Add Users': 
             - 'Overview': guides/user-mgt/add-users-overview.md
             - 'Add Users and Assign Password': guides/user-mgt/add-user.md
-            - 'Add Users and Ask Password': guides/user-mgt/ask-password.md
+            - 'Add Users and Invite': guides/user-mgt/ask-password.md
```

Comment:
should this be "Invite Users"? what did we finally decide to call this?

Reply:
We decided on Add Users and Invite

Reply:
can we make it Add and Invite Users?

Reply:
+1. Will correct in my next PR.